mozilla-release/mobile/android/chrome/content/browser.js

```diff
@@ -6892,6 +6893,11 @@ var Cliqz = {
           name: 'updateBlocking',
           message: blockingPolicy
         });
+        break;
+      case "Search:ChangeTheme":
+        const theme = data.isLightTheme ? "white" : "blue";
```

Comment:
Please edit this line to let extension know of the selected theme at startup time:
https://github.com/ghostery/browser-android/blob/eaea5f79164c2567f8460f1baffed5d1d339da48/mozilla-release/mobile/android/chrome/content/browser.js#L6645
please append url to cards with background parameter: `?bg=${theme}`
while theme is either `white` or `blue`

Reply:
Hey, @khaled-cliqz I did that. Please check the new commit